packages/dds/merge-tree/src/partialLengths.ts

```diff
@@ -1138,19 +1197,19 @@ export function verifyExpectedPartialLengths(
 
 	let expected = 0;
 	const nodesToVisit: IMergeNode[] = [node];
+	const perspective =
```

Comment:
besides constructing the same perspective less within the while loop, this is practically useful for debugging issues (usually you want to drop a break point inside the block where we find expected !== actual, and at that point it's convenient to have this perspective in a local to play around with)